frontend/.eslintrc.js

```diff
@@ -108,11 +112,15 @@ module.exports = {
             // Turn off some intrusive rules
             "react/prop-types": "off",
             "react/react-in-jsx-scope": "off",
+            "react/display-name": "off",
             "@typescript-eslint/no-explicit-any": "off",
             "@typescript-eslint/no-use-before-define": "off",
             "@typescript-eslint/typedef": "off",
             "@typescript-eslint/prefer-readonly-parameter-types": "off",
             "@typescript-eslint/no-magic-numbers": "off",
+            "@typescript-eslint/explicit-function-return-type": "off",
+            "@typescript-eslint/promise-function-async": "off",
+            "@typescript-eslint/space-before-function-paren": "off",
```

Comment:
Why turn this off?

Reply:
This would enforce this:

```
public constructor (msg: string)
```

Do we really want this space? Never seen that style before. And we also don't use it for normal functions.

Reply:
Ui, okay? Weird default. I would say we want to enforce **not** having the space. Can we configure that rule?